frontend/lambda-in/src/Components/DevInfoEditz/DevInfoEditz.js

```diff
@@ -1,38 +1,221 @@
+    this.update = this.update.bind(this);
+  }
+
+  /**
+   * Get a copy of user's data to keep global state inmutable.
+   */
+  componentDidMount() {
+    // eslint-disable-next-line react/prop-types
+    const { getGS } = this.props;
+    // // console.log('FORM', this.props);
+    this.userStateCopy = { ...getGS('userInfo') };
+    this.setState({ ready: true, ...getGS('userInfo') });
+
+    /**
+     * Listen for MapChip's custom event emmited when a item is deleted.
+     */
+    // eslint-disable-next-line react/no-find-dom-node
+    ReactDOM.findDOMNode(this).addEventListener(
+      'onDeleteItem',
+      this.handleOnDeleteItem,
+    );
+
+    // eslint-disable-next-line react/no-find-dom-node
+    ReactDOM.findDOMNode(this).addEventListener(
+      'onCreateItem',
+      this.handleCreateItem,
+    );
+  }
+
+  componentWillUnmount() {
+    // eslint-disable-next-line react/no-find-dom-node
+    ReactDOM.findDOMNode(this).removeEventListener('onDeleteItem', () => null);
+    // eslint-disable-next-line react/no-find-dom-node
+    ReactDOM.findDOMNode(this).removeEventListener('onCreateItem', () => null);
+  }
+
+  /**
+   * Update local state.
+   *
+   * @method setFormState
+   * @param {object} properties - Properties to be set. { property: value }
+   * @return {void}
+   *
+   * @example Pass as a prop to component.
+   * <Component setLS={this.setFormState} />
+   */
+  setFormState(properties) {
+    // // console.log({ setGS: properties });
+    this.setState(properties);
+  }
+
+  /**
+   * Sync local state with input field.
+   */
+  handleChange(event) {
+    // // console.log('Form Dev update');
+    event.stopPropagation();
+    this.setState({ [event.target.id]: event.target.value });
+  }
+
+  updateArray(dataset) {
+    /**
+     * Data attributes expected
+     * {
+     *  'data-field': Main field,
+     *  'data-itemtype': Type of data the array contain,
+     *  'data-index': index,
+     *  'data-property': If 'itemtype' === 'object -> this define the object's 'property',
+     *  'data-value': New value,
+     * }
+     */
+    // console.log('FORM updateArray DATASET', dataset);
+    // prettier-ignore
+    const {
+      field,
+      itemtype,
+      index,
+      property,
+      subindex,
+      value,
+    } = dataset;
+
+    switch (itemtype) {
+      case 'object':
+        // console.log('FORM updateArray [Array][Object]', {
+          index,
+          property,
+          value,
+        });
+        this.userStateCopy[field][index][property] = value;
+        break;
+      case 'array':
+        // console.log('FORM updateArray [Array][Array]', {
+          index,
+          subindex,
+          value,
+        });
+        break;
+      // If itemSchema='singleItem'
+      default:
+        // '[object String]' || '[object Number]' || // '[object Boolean]'
+        // console.log('FORM updateArray [Array][String]', {
+          field,
+          itemtype,
+          value,
+        });
```

Comment:
Same here. I think you meant to comment out multiple lines. As is this is throwing an error.